mne/viz/_3d.py

```diff
@@ -499,6 +521,41 @@ def plot_trans(info, trans='auto', subject=None, subjects_dir=None,
     if not isinstance(info, Info):
         raise TypeError('info must be an instance of Info, got %s'
                         % type(info))
+    if not (source is None) ^ (bem is None):
+        raise ValueError('source must be None if bem is not None '
+                         '(and vice versa)!')
+    is_sphere = False
+    if isinstance(bem, ConductorModel) and bem['is_sphere']:
+        if len(bem['layers']) != 4:
+            raise ValueError('The sphere conductor model must have three '
+                             'layers for plotting.')
+        is_sphere = True
```

Comment:
Actually we could do something simple for MEG. Maybe just an opaque tiny (0.5 cm radius?) sphere at the origin would work to indicate only the origin matters.